pkgs/development/python-modules/openai-whisper/default.nix

```diff
@@ -50,11 +51,13 @@ buildPythonPackage rec {
     numba
     scipy
     tiktoken
-  ] ++ lib.optionals (!cudaSupport) [
+  ] ++ lib.optionals (!cudaSupport && !rocmSupport) [
     torch
   ] ++ lib.optionals (cudaSupport) [
     openai-triton
     torchWithCuda
+  ] ++ lib.optionals (rocmSupport) ]
+    torchWithRocm
```

Comment:
``torchWithRocm`` is not in the arguments above.
Like the previous comment, I suggest doing the aforementioned instead.